src/cls/rgw/cls_rgw_types.h

```diff
@@ -1224,6 +1245,9 @@ struct cls_rgw_gc_obj_info
     chain.dump(f);
     f->close_section();
     f->dump_stream("time") << time;
+  }
+  void decode_json(JSONObj *obj){
+    
```

Comment:
@iqbalredkhan is there supposed to be something in the body of this function?

Reply:
I had created another PR with all required changes and addition of functions
https://github.com/ceph/ceph/pull/45394